ethereum/core/src/test/java/org/hyperledger/besu/ethereum/forkid/ForkIdTestUtil.java

```diff
@@ -79,6 +84,13 @@ public static class Forks {
     public static final List<Long> SHANDONG =
         Arrays.asList(0L, 0L, 0L, 0L, 0L, 0L, 0L, 0L, 0L, 0L, 0L);
     public static final List<Long> PRIVATE = Arrays.asList(0L, 0L, 0L, 0L, 0L, 0L, 0L, 0L);
+
+    public static final List<Long> WITHDRAWALS_BLOCKS =
```

Comment:
s/WITHDRAWALS/SHANGHAI ?

Reply:
Think WITHDRAWALS makes more sense as the rest are are network names not fork names

Reply:
Why is withdrawals more like a network than shanghai? I think it's just one of several features within shanghai 🤷

Reply:
Because withdrawals in this case represents the withdrawals testnet. Shanghai is the fork so don't want to use that. But could just sidestep this and put these into the mainnet configuration instead.